server/src/main/java/com/linecorp/centraldogma/server/CentralDogma.java

```diff
@@ -371,7 +375,8 @@ private void doStart() throws Exception {
 
     private CommandExecutor startCommandExecutor(
             ProjectManager pm, Executor repositoryWorker, MeterRegistry meterRegistry,
-            @Nullable SessionManager sessionManager) {
+            @Nullable SessionManager sessionManager, StoragePurgeService storagePurgeService) {
```

Comment:
How about sort the parameters in the order of importance, with an optional one placed at last, i.e.

- `ProjectManager`
- `Executor`
- `StoragePurgeService`
- `MeterRegistry`
- `SessionManager`

?

Reply:
I tended to preserve the order of the existed parameters.
But this method is a private and "an optional one placed at last" is reasonable. Thanks